src/object.c

```diff
@@ -1434,7 +1434,7 @@ NULL
         addReplyVerbatim(c,info,sdslen(info),"txt");
         sdsfree(info);
 #else
-        addReplyBulkCString(c,"Stats not supported for the current allocator");
+        addReplyError(c,"Stats not supported for the current allocator");
```

Comment:
Not sure if it's worth it to do this change. This can be either an error or a just an info string. Both are fine.

Reply:
oh... it’s actually a bit embarrassing.
In this case, only the comma is left to add .. like https://github.com/redis/redis/pull/9153/commits/dca1346715fc0dded2c84854d6d2ca0a447ac4ea
quite small pr..

Reply:
Don't worry, it is what it is. I suggest fixing this PR (and it's title) accordingly.
Generally changes that don't make a difference should be avoided so not to dirty the blame log. That's why I suggest leaving it as `addReplyBulkCString()`. The command should be fixed.

Reply:
I see, thank you. Alerady Updated